src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -138,6 +149,10 @@ public int order() {
             apply0(task, action, request, listener, chain);
         }
     }
+
+    private static Set<String> alias2Name(Set<Alias> aliases) {
+        return aliases.stream().map(a -> a.name()).collect(ImmutableSet.toImmutableSet());
```

Comment:
Should we add a null check for "aliases" ?

Reply:
It assumes not `null` `aliases` that is always true for `CreateIndexResponse.aliases()`.